packages/synthetics-sdk-broken-links/package.json

```diff
@@ -40,6 +40,7 @@
   },
   "dependencies": {
     "@google-cloud/synthetics-sdk-api": "google-cloud-synthetics-sdk-api-0.1.0.tgz",
+    "plimit-lit": "^1.5.0",
```

Comment:
why is this added?

Reply:
This is what allows to limit concurrency (see `checkLinks`)

My comment from above that probably got lost in the ether:

https://github.com/GoogleCloudPlatform/synthetics-sdk-nodejs/pull/41#discussion_r1320365682